codebuild/spec/buildspec_general_batch.yml

```diff
@@ -0,0 +1,116 @@
+---
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"). You may not use
+# this file except in compliance with the License. A copy of the License is
+# located at
+#
+# http://aws.amazon.com/apache2.0/
+#
+# or in the "license" file accompanying this file. This file is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or
+# implied. See the License for the specific language governing permissions and
+# limitations under the License.
+version: 0.2
+
+batch:
+  build-list:
+    # Saw
+    - identifier: s2nSawBike
+      buildspec: codebuild/spec/buildspec_ubuntu.yml
+      env:
+        privileged-mode: true
+        compute-type: BUILD_GENERAL1_2XLARGE
+        variables:
+          S2N_LIBCRYPTO: openssl-1.0.2
+          TESTS: sawBIKE
+          SAW: true
+          GCC_VERSION: NONE
+
+    - identifier: sawHMACPlus
+      buildspec: codebuild/spec/buildspec_ubuntu.yml
+      env:
+        privileged-mode: true
+        compute-type: BUILD_GENERAL1_LARGE
+        variables:
+          TESTS: sawHMACPlus
+          SAW: true
+          GCC_VERSION: NONE
+
+    - identifier: s2nSawTls
+      buildspec: codebuild/spec/buildspec_ubuntu.yml
+      env:
+        privileged-mode: true
+        compute-type: BUILD_GENERAL1_LARGE
+        variables:
+          TESTS: tls
+          SAW: true
+          GCC_VERSION: NONE
+
+    # Other
+    - identifier: s2nSidetrail
+      buildspec: codebuild/spec/buildspec_sidetrail.yml
+      env:
+        privileged-mode: true
+        compute-type: BUILD_GENERAL1_2XLARGE
+        image: 024603541914.dkr.ecr.us-west-2.amazonaws.com/linux-docker-images:sidetrail
```

Comment:
Do we want to keep this as a hardcoded value?

Reply:
yes, this is actually an URL, the full version has `docker://` prepended.